### PR TITLE
Report 100% stability if no unstable edges are found

### PR DESCRIPTION
**Pull request by sadeli413**
This adds extra behavior to `CalibrationStage`, where on the first run if there are no unstable edges, it will report 100% stability. Addresses #2214

### CODE REVIEW

File: libafl/src/stages/calibrate.rs [hunk 1]
```diff
@@ -71,6 +71,8 @@ pub struct CalibrationStage<C, O, OT, S> {
     stage_max: usize,
     /// If we should track stability
     track_stability: bool,
+    /// If we should send the default stability
+    need_default_stability: bool,
```
Comment (domenukk) [inline libafl/src/stages/calibrate.rs:75]: This needs to be part of the metadata, else the stage will always resend 100% after a crash.
Probably the best way would be to always create the `UnstableEntriesMetadata`, and then only send 100% if it didn't exist before - so use its existence as flag

Comment (domenukk) [reply libafl/src/stages/calibrate.rs:75]: (explanation: on crash the InProcessExecutor will restart/refork the program and reload the state - anything that's not in the state (metadtata lives in the state) will be reset to the initial value, accordingly

**Comment by domenukk** [comment]
Thank you for your PR! Let me know if I should take a stab at it, else feel free to address my comments :)

**Comment by domenukk** [comment]
@sadeli413 please take a look at #2217, I had some time on my hands..

**Comment by sadeli413** [comment]
> @sadeli413 please take a look at #2217, I had some time on my hands..

Just saw this, thanks!

**Comment by domenukk** [comment]
Sorry for the duplicate work 😬

**Pull request closed without merging**